metsrw/metadata.py

```diff
@@ -334,9 +335,14 @@ def get_status(self):
             return self.status
         if self.subsection == "dmdSec":
             if self.older is None:
-                return "original"
+                status = "original"
+                if self.newer is not None:
+                    status += "-superseded"
             else:
-                return "updated"
+                status = "update"
+                if self.newer is not None:
+                    status += "-superseded"
+            return status
```

Comment:
@replaceafill ... I kept option to use the `replace_with` function with dmdSecs and updated this status values to match the new ones in `add_dmdsec`. However, the updated `add_dmdsec` function takes care of this so I'm not sure what would be the best approach for `replace_with`, maybe we could deprecate it somehow for dmdsecs? We are also making quite a change on the status attributes so we should think a little about the release of this modifications. Thoughts?